config.yml

```diff
@@ -5,6 +5,7 @@ scenario_params_list:
    amounts_per_node: [0.4, 0.3, 0.3] 
    samples_split_option: 'Random'
    single_partner_test_mode: 'global'
+   epoch_count:4
```

Comment:
Need a space after the colon **:** otherwise it crashes when parsing.
Also can you please add it to all scenario on this config file (to be consistent), and make it around 40. (4 is a bit small!) 
Thanks :)

Reply:
Done ! :)